README.md

```diff
@@ -37,43 +37,82 @@ npm run dev
 
 ## Running [rclone](https://rclone.org) against Permanent.org instances
 
-- Install rclone, using [official install guide](https://rclone.org/install/)
+First install rclone.  See the [official installation
+guide](https://rclone.org/install/) for more information.
 
-### Setting up an rclone remote
+### Set up an rclone remote
 
-- `rclone config`
-- Select option n for new and enter a remote name such as `permanent-prod` or `permanent-dev`.
-- Set `SFTP` as storage protocol by by selecting from displayed menu, after name prompt.
-- Set an ssh host, the host format is `sftp.${environment}.permanent.org` where `${environment}` is either `dev` or `staging` for the dev and staging environments respectively.
-    - For example `sftp.dev.permanent.org` (for dev).
-    - Prod has ssh host simply as `sftp.permanent.org` (without specifying environment).
-- Set your permanent username once prompted for example `rclone.joy@example.com`.
-- Set `SFTP` port as `22` which is the default.
-- Set `ssh` password by selecting the "Yes, type in my own password" (`y`) option, when the password options show up.
-- Set your remote password which is the password to your permanent account (in this case `rclone.joy@example.com`).
-- Set every other option from this point to the end, with its default value by simply pressing enter.
+Run **`rclone config`** and answer the questions it asks:
 
-At the end of the configuration you should have something like this:
+- Select option **`n`** (for "new") and make a remote name (e.g., "permanent").
+
+- At the **`Storage>`** prompt, set the storage protocol to SFTP, either by typing "sftp" at the prompt or by selecting the corresponding number from the displayed menu (the numbered entry may say "SSH/SFTP").
+
+- At the **`host>`** prompt, set the host to **`sftp.permanent.org`**.
+
+  If you want to run rclone against the dev or staging environment, then use `sftp.dev.permanent.org` or `sftp.staging.permanent.org` instead.  (You can also say `sftp.prod.permanent.org`, but it means the same thing as just `sftp.permanent.org`.)
```

Comment:
> (You can also say `sftp.prod.permanent.org`, but it means the same thing as just `sftp.permanent.org`.)

I wonder if this is true.

Reply:
Well, I'm merely repeating a claim that was in the original.

Reply:
> I wonder if this is true.

It is not true -- I misread some material about dev and staging in the original text, and mistakenly applied it to prod as well.  Thanks for catching this!  I've pushed the fix (commit e1e92104712) to this branch.